### ISSUE TITLE
Get representative sequences - 16S arch/bac

### ISSUE THREAD

**Issue by hollybik** [gjospin/PhyloSift#12]

**Comment by gjospin** [comment]
Using the pruned tree provided by Aaron (@koadman), I found sequences that were not labeled as Bacteria or Archaea in the gg taxonomy. They were not found in the gg unaligned fasta file.  Should I grab them from ncbi anyways ?
Here is an example of such sequence. http://www.ncbi.nlm.nih.gov/nuccore/DQ837295

Additionally, I found some sequences from the pruned tree that were not in the gg unaligned file BUT labeled as Archaea or Bacteria in the gg taxonomy.  Some of those sequences came back as such from the ncbi website:
http://www.ncbi.nlm.nih.gov/nuccore/287794
